### PR TITLE
DOC-2510: Added v20.2.19 release notes

### PR DESCRIPTION
**Pull request by nickvigilante**
Fixes DOC-2510

### CODE REVIEW

File: releases/v20.2.19.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+{% include_cached copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach:v20.2.19
+~~~
+
+### Bug fixes
+
+- [Schema changes](../v20.2/online-schema-changes.html) running during node shutdown could sometimes fail permanently when they should not. [#71557][#71557]
+- Fixed a bug which prevented the Data Distribution page from working on clusters which were upgraded from 19.2 or earlier. [#72506][#72506]
+- The CancelSession endpoint now correctly propagates gateway metadata when forwarding requests. [#75885][#75885]
```
Comment (gemma-shay) [inline releases/v20.2.19.md:40]:
```suggestion
- The `CancelSession` endpoint now correctly propagates gateway metadata when forwarding requests. [#75885][#75885]
```

**Review by gemma-shay — APPROVED** [review]
LGTM besides the one comment, but I've never done CRDB release notes before so I'm going to add Lauren to review.

**Review by lnhsingh — COMMENTED** [review]
Couple updates. Once fixed and the binaries are blessed, feel free to publish and alert the appropriate comms channels

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/12957#-:-MvU-hLR42rYSyIgl1Hc:bxxpdkl)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @celiala and @nickvigilante)

___
*[releases/v20.2.19.md, line 7 at r2](https://reviewable.io/reviews/cockroachdb/docs/12957#-MvU-Crs3N3DzJ7dT08Y:-MvU-Crt2JmPCWpE4tEP:b-rcptr6) ([raw file](https://github.com/cockroachdb/docs/blob/0db9311cffd716ed9fc02f18e9afbcbddabfa3c4/releases/v20.2.19.md#L7)):*
> ```Markdown
> ---
> 
> ## February 07, 2022
> ```

Make sure this date is updated to today

(Also, I think the norm is February 9, 2022 - without the 0 in front)
___
*[releases/v20.2.19.md, line 38 at r2](https://reviewable.io/reviews/cockroachdb/docs/12957#-MvU-Zb15gQEo5OOGcgz:-MvU-Zb15gQEo5OOGch-:bx4wdb5) ([raw file](https://github.com/cockroachdb/docs/blob/0db9311cffd716ed9fc02f18e9afbcbddabfa3c4/releases/v20.2.19.md#L38)):*
> ```Markdown
> ### Bug fixes
> 
> - [Schema changes](../v20.2/online-schema-changes.html) running during node shutdown could sometimes fail permanently when they should not. [#71557][#71557]
> ```

Change to:
```
Previously, [Schema changes](../v20.2/online-schema-changes.html) running during node shutdown could sometimes fail permanently when they should not. This is now fixed. [#71557][#71557]
```



_Code quote:_
```Markdown
[Schema changes](../v20.2/online-schema-changes.html) running during node shutdown could sometimes fail permanently when they should not. [#71557][#71557]
```

**Review by nickvigilante — COMMENTED** [review]
Done. I posted a question in #docs-style asking about the leading 0.

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/12957#-:-MvU3uFOA9An_ihwAYUF:bx0g55u)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @celiala, @gemma-shay, and @lnhsingh)

___
*[releases/v20.2.19.md, line 40 at r1](https://reviewable.io/reviews/cockroachdb/docs/12957#-MvK5uAj1WgwwNauv92B-r1-40:-MvU3t4i8HTYQD2jCQCd:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/71765ea29a67a3ec75435b71941320e78ebd2eb4/releases/v20.2.19.md#L40)):*
<details><summary><i>Previously, gemma-shay (Gemma Shay) wrote…</i></summary><blockquote>

```suggestion
- The `CancelSession` endpoint now correctly propagates gateway metadata when forwarding requests. [#75885][#75885]
```
</blockquote></details>

Done.
___
*[releases/v20.2.19.md, line 7 at r2](https://reviewable.io/reviews/cockroachdb/docs/12957#-MvU-Crs3N3DzJ7dT08Y:-MvU3ltM5EraviBb0Smx:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/0db9311cffd716ed9fc02f18e9afbcbddabfa3c4/releases/v20.2.19.md#L7)):*
<details><summary><i>Previously, lnhsingh (Lauren Hirata Singh) wrote…</i></summary><blockquote>

Make sure this date is updated to today

(Also, I think the norm is February 9, 2022 - without the 0 in front)
</blockquote></details>

Done.
___
*[releases/v20.2.19.md, line 38 at r2](https://reviewable.io/reviews/cockroachdb/docs/12957#-MvU-Zb15gQEo5OOGcgz:-MvU3nDGBpZXaAwvIXh3:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/0db9311cffd716ed9fc02f18e9afbcbddabfa3c4/releases/v20.2.19.md#L38)):*
<details><summary><i>Previously, lnhsingh (Lauren Hirata Singh) wrote…</i></summary><blockquote>

Change to:
```
Previously, [Schema changes](../v20.2/online-schema-changes.html) running during node shutdown could sometimes fail permanently when they should not. This is now fixed. [#71557][#71557]
```
</blockquote></details>

Done.

**Pull request merged**